hadoop/src/test/java/org/apache/iotdb/hadoop/tsfile/TSFHadoopTest.java

```diff
@@ -38,7 +39,7 @@
 
   private TSFInputFormat inputFormat = null;
 
-  private String tsfilePath = "example_mr.tsfile";
+  private String tsfilePath = TestConstant.BASE_OUTPUT_PATH.concat("example_mr.tsfile");
```

Comment:
Actually I think "+" is enough for strings concat... Do you think it is necessary to change it to be consist with other place in the project?  @qiaojialin 
```suggestion
  private String tsfilePath = TestConstant.BASE_OUTPUT_PATH + "example_mr.tsfile";
```